CMakeLists.txt

```diff
@@ -333,11 +333,12 @@ if (FLANG_INCLUDE_TESTS)
 endif()
 
 # TODO: Add doxygen support.
-#option(FLANG_INCLUDE_DOCS "Generate build targets for the Flang docs."
-#       ${LLVM_INCLUDE_DOCS})
-#if (FLANG_INCLUDE_DOCS)
-#  add_subdirectory(documentation)
-#endif()
+#add_custom_target(doxygen-flang)
+option(FLANG_INCLUDE_DOCS "Generate build targets for the Flang docs."
+	ON) #currently explicitely ON, else depends on ${LLVM_INCLUDE_DOCS} 
```

Comment:
nit typo: explicitly

Reply:
I shall add `LLVM_INCLUDE_DOCS ` as a dependency, and remove comments above.
Is this probably why @tskeith sees the error.
```

CMake Error at docs/cmake_install.cmake:36 (file):
  file INSTALL cannot find
  "<build-dir>/docs/doxygen/html".
Call Stack (most recent call first):
  cmake_install.cmake:48 (include)

```

Reply:
Added dependency on `LLVM_INCLUDE_DOCS`